documentation/modules/exploit/unix/webapp/joomla_sqli_rce_3_7_0.md

```diff
@@ -0,0 +1,49 @@
+## Vulnerable Application
+
+  This module exploits a sql injection in the core of Joomla 3.7.0.
+  This vulnerability can allow remote code execution.
+
+## Verification
+
+
+  1. Start msfconsole
+  2. Do: `use exploit/unix/webapp/joomla_comfields_sqli_rce`
+  3. Do: `set rhost [ip]`
+  4. Do: `set tageturi [uri]`
+  5. Do: `exploit`
+  6. Get a shell
+
+## Scenarios
+
+### Joomal 3.7.0 and an administrator must be authenticated in the backend
```

Comment:
Joomla 3.7.0 on Ubuntu 16.04 with another user authenticated as an administrator

Reply:
In the comments you mention Super User, here you mention administrator.  I'm not familiar with Joomla roles, but I'd assume there is a difference?